app/routers/quizzes.py

```diff
@@ -92,6 +93,75 @@ async def get_quiz(quiz_id: str):
     quiz_collection = client.quiz.quizzes
     if (quiz := quiz_collection.find_one({"_id": quiz_id})) is not None:
         update_quiz_for_backwards_compatibility(quiz_collection, quiz_id, quiz)
+
+        if "metadata" in quiz and quiz["metadata"] is not None:
+            if "quiz_type" in quiz["metadata"]:
```

Comment:
So we're assuming that if quiz is not `omr-type`, then assume it's normal quiz? Makes sense

Reply:
yes